### ISSUE TITLE
Smart App Control on my windows 11 desktop is blocking ElevenClock

### ISSUE THREAD

**Issue by da-zel** [marticliment/ElevenClock#1117]
## Before submitting the issue

- [x] **I have checked that the bug is not a duplicate or, if it is, it needs to be reopened**
- [x] **I have checked that this issue is not present in the [Known Issues](https://github.com/martinet101/ElevenClock/wiki#known-issues) section on the wiki.**
- [x] **I have checked that I am running the latest version of ElevenClock**
- [x] **I have tried resetting and reinstalling ElevenClock and the issue persisted or happened again**
- [x] **I have checked the [Common Problems](https://github.com/martinet101/ElevenClock/wiki#common-problems) section on the wiki for solutions**

</b>

## To Reproduce
Steps to reproduce the behaviour:

## The Log

```
Please paste here the log (Go to ElevenClock Settings -> Scroll to the bottom -> Open ElevenClock Log. 

If the issue can be reproduced (As an example: After waiting 5 minutes, the clock shows a wrong format), please 
send the log AFTER reproducing the issue (1. Launch ElevenClock, 2. Reproduce the issue, 3. Paste the log)
```

**Comment by marticliment** [comment]
This happens because ElevenClock isn't signed. This is listed in the known issues section

**Comment by da-zel** [comment]
Thank you for responding.  I signed up to Github just to post my issue.
Due to my general ignorance, I got a bit overwhelmed and failed to find the
previous posting, though I did try.   But I went back just now, looked
again, and found it. So I now understand the situation.  I was/had been
baffled by the block because I had assumed (ignorance again) that being in
the Microsoft Store meant an app had already been vetted.  And indeed, when
I first installed your app, it ran for a month before Smart App Control
blocked it.  Then, thinking this was some kind of glitch, I reinstalled it
and it ran for another week before being blocked again.  Now I understand,
from your response to the previous post, that the 'quasi-requirement' for
signing an app (quasi-enforced by blocking) which Microsoft claimed and
explained in the bits I read where they wrote about this was all about
security, is really more about money (big surprise).

Now I can close with some good news.  Yesterday I submitted feedback to
Microsoft about this issue and just now, in the midst of writing this, I
tried opening ElevenClock again and it's back.  No block message.  Let's
hope this time it's permanent.

Cheers, I am very h-app-y with your app
